functions/lib/sql-helper.js

```diff
@@ -26,3 +26,25 @@ module.exports.sqlFindCheckByCheckCode = async function (checkCode) {
   const res = await sqlService.query(sql, params)
   return R.head(res)
 }
+
+/**
+ * Retrieve all pupil checks based on check code
```

Comment:
More sense to me: 'Find all checks for a particular pupil based on one of the pupil's `checkCodes`'

Reply:
@jon-shipley well said, updated.